src/lib/stats/enchanting.ts

```diff
@@ -0,0 +1,55 @@
+import type { Experimentation, SkyBlockProfileMember, SkyblockExperimentTier } from '$types/hypixel';
+import { EXPERIMENTS } from '$constants';
+import _ from 'lodash';
+
+export function getEnchanting(userProfile: Partial<SkyBlockProfileMember>) {
+	if (!userProfile.experimentation) return;
+
+	return {
+		unlocked: !!userProfile.experimentation,
+		experiments: userProfile.experimentation
+			? Object.fromEntries(
```

Comment:
This whole `Object.fromEntries(...)` call should be separated into a new function, it's doing way too much to be inlined like this. I know we're kinda doing the same thing elsewhere already, but we should probably be more strict with things like this